source/developers/experience-builder.rst

```diff
@@ -1248,6 +1253,8 @@ Usage
 XB JS libraries can be used either via npm by importing ``@craftercms/experience-builder`` or using the
 JS umd bundle and adding it into your app's runtime.
```

Comment:
What's JS umd?

Reply:
A term from the JS community/ecosystem: [Universal Module Definition](https://riptutorial.com/javascript/example/16339/universal-module-definition--umd-)